### ISSUE TITLE
question:  why do you prefer bash over zsh ?

### ISSUE THREAD

**Issue by gaboesquivel** [alrra/dotfiles#23]
Hi alrra, your dotfiles repo looks great.  
I've been using oh-my-zsh for a while and now I want something more personalized and efficient, so I'm considering options. I would like to know your reasons to prefer bash over zsh.

**Comment by alrra** [comment]
@gaboesquivel I do apologise for the delayed answer. Somehow I have missed the notification for this issue, and afterwards I forgot about it.

---

> your reasons to prefer bash over zsh.

Note: I have tried `Zsh` (and `oh-my-zsh`) a few years back, therefore some of the points below may be outdated at this point. 

* `Zsh` has some good features over `Bash`, first class arrays being one that I really liked. But, in general, most of them seemed to me like "nice to have", and overall, not enough to make me feel a need to switch. Plus, recent versions of `Bash` started introducing some of the "nice to have" features that existed in `Zsh` (e.g.: recursive globbing).

   Also, when I've tried `Zsh`, its `emulate` feature was kinda bad, and switching to `Zsh` as my default shell made some things stop working properly.

* `Bash` is everywhere ([now, even on Windows!! 😮](https://msdn.microsoft.com/commandline/wsl/about)). I have yet to encounter a Unix-like OS that I needed to work with that didn't came with `Bash`. And this is a huge plus as far as I'm concerned as it allowed me countless times to reuse `Bash` scripts that I've written throughout the years without any or with little modifications.
